app/src/main/java/org/oppia/app/utility/RegionClickEvent.kt

```diff
@@ -0,0 +1,8 @@
+package org.oppia.app.utility
+
+/** Sealed class containing the items for accessibility and non-accessibility use-case to in conjunction with [ClickableAreasImage]. */
+sealed class RegionClickedEvent
+
+data class NamedRegionClickedEvent(val regionLabel: String) : RegionClickedEvent()
```

Comment:
This & the next class also should have KDocs explaining when they'll be used. _All_ public-API classes need to have KDocs. This is actually really important--documentation provides necessary context for other team members who'll be reading this code. Without that context, future investigations and code changes may be much harder and more error-prone.

Reply:
done